### PR TITLE
udata3 compatibility

### PR DESCRIPTION
**Pull request by quaxsze**
Depends on https://github.com/opendatateam/udata/pull/2643

### CODE REVIEW

**Comment by abulte** [comment]
## Dataset

Capture d’écran 2021-08-09 à 13 48 13

## Reuse

Capture d’écran 2021-08-09 à 13 50 26

_NB: style is the same as recommended APIs_

Capture d’écran 2021-08-09 à 13 50 43

File: udata_recommendations/templates/dataset-recommendations.html [hunk 1]
```diff
@@ -9,9 +9,9 @@ <h3>Réutilisations recommandées</h3>
             </span>
         </div>
     </div>
-    <div id="dataset-recommendations-container" class="card-list card-list--columned">
+    <div id="dataset-recommendations-container" class="py-md">
         {% for reuse in reco_reuses %}
-            <div class="col-xs-12 col-md-6"
+            <div class="card reuse-card border p-sm unstyled w-100 mb-md"
```
Comment (maudetes) [inline udata_recommendations/templates/dataset-recommendations.html:14]: Not convinced about this style, as it doesn't seem consistent with the vertical reuse card layout. Maybe use the same classes as [here](https://github.com/etalab/udata-gouvfr/blob/master/udata_gouvfr/theme/gouvfr/templates/dataset/display.html#L365).

Comment (abulte) [reply udata_recommendations/templates/dataset-recommendations.html]: I changed to style to have the best of both horizontal and vertical worlds :p

**Review by maudetes — APPROVED** [review]
Looks good to me for a first iteration :)

**Comment by abulte** [comment]
@maudetes yay tests passing 🎉 I'll merge when udata is released with the correct requirement.

**Pull request merged**